dependency-check-suppressions.xml

```diff
@@ -40,5 +40,7 @@
         <cve>CVE-2019-11358</cve>
         <cve>CVE-2020-11022</cve>
         <cve>CVE-2020-11023</cve>
+        <cve>CVE-2020-15824</cve>
```

Comment:
We don't want to be adding more suppressions if we can help it. Where are these coming from please?

Reply:
My initial build failed due to a dependency issue and I saw that it was fixed with this suppression which I saw in PR1918 and which is now merged with master.

https://github.com/hmcts/sscs-tribunals-case-api/pull/1918/commits/b80a0b5768d9ea5c0cb5921e6ec18249e70ed02a